roles/council-representative.md

```diff
@@ -0,0 +1,48 @@
+# Role Description: Rust Council Representative
+
+This is a description of the role and responsibilities for a Rust Council Representative.
+Representatives should read and be familiar with the [Policies governing the Leadership Council][forge-lc].
+
+[forge-lc]: https://forge.rust-lang.org/governance/council.html
+
+## Goals
+
+- Represent the needs and interests of the team and subteams you represent within the Council.
+- Represent the needs of the Rust Project at large within the team and subteams you represent.
+- Ensure effective communication, collaboration, and coordination among teams.
+- Work towards the long-term success of the Rust Project.
+
+## Time Budget
+
+- 2-5 hours per week
```

Comment:
```suggestion
- 2-5 hours per week typically
  - Sometimes more time can be needed, even up to 20 hours a week.
```

I think 2-5 hours underestimates the time needed, or at least the time I put in. For me, a typical week is probably 2 hours, maybe even one hour, but there have also been periods when council work was almost full time (for example, during Project Director elections).